tensorboard/plugins/debugger/README.md

```diff
@@ -0,0 +1,175 @@
+# The Debugger Dashboard
+
+The debugger dashboard offers a graphical user interface for the [TensorFlow debugger](https://www.tensorflow.org/programmers_guide/debugger). For instance, this dashboard enables users to
```

Comment:
Optional ProTip: Did you know it's possible to do hyperlinks in markdown where you punt the long ugly URLs to after the section, or to the very bottom entirely?